documentation/modules/proc-uninstalling-using-olm.adoc

```diff
@@ -52,7 +52,7 @@ endif::[]
 {cmdcli} delete crd -l app=enmasse
 ----
 
-. (Optional. Skip this step if {ProductName} is installed in the {OperatorNamespace} namespace) Delete the namespace where {ProductName} was installed:
+. (Optional. Skip this step if {ProductName} is installed in the {OperatorName} namespace) Delete the namespace where {ProductName} was installed:
```

Comment:
@lulf: Now I'm thinking this should actually be `{OperatorSourceName}`?

Reply:
No, I think its just a missing attribute. It should be set to 'openshift-operators' for openshift, and 'operators' for kubernetes.